docs/vars.md

```diff
@@ -105,6 +126,25 @@ following default cluster parameters:
   converted to RBAC mode. However, your apps which rely on Kubernetes API will
   require a service account and cluster role bindings. You can override this
   setting by setting authorization_modes to `[]`.
+  
+* *kube_apiserver_admission_control_config_file* - Enable configuration for `kube-apiserver` admission plugins.
+  Currently this variable allow you to configure the `EventRateLimit` admission plugin.
+  
+  To configure the **EventRateLimit** plugin you have to define a data structure like this:
+
+```yml
+event_rate_limits:
```

Comment:
Sorry for missing this point on my previous review.
Could you rename this option to `kube_apiserver_admission_event_rate_limits` to understand that is an admission related option easily? Because Kubespray contains a lot of options.

Reply:
Sure

Reply:
Done